api/capi/include/nntrainer-tizen-internal.h

```diff
@@ -0,0 +1,74 @@
+// SPDX-License-Identifier: Apache-2.0
+/**
+ * Copyright (C) 2021 Jihoon Lee <jhoon.it.lee@samsung.com>
+ *
+ * @file   nntrainer-tizen-internal.h
+ * @date   30 June 2021
+ * @brief  NNTrainer CAPI header for tizen interanl api.
+ * @note   This header is designed to be used only in Tizen
+ * @see    https://github.com/nnstreamer/nntrainer
+ * @author Jihoon Lee <jhoon.it.lee@samsung.com>
+ * @bug    No known bugs except for NYI items
+ *
+ */
+
+#ifndef __NNTRAINER_TIZEN_INTERNAL_H__
+#define __NNTRAINER_TIZEN_INTERNAL_H__
+
+#include <nntrainer.h>
+
+#ifdef __cplusplus
+extern "C" {
+#endif /* __cplusplus */
+
+/**
+ * @brief Save the model after training
+ * @details Use this function to save the current model. @a format rules
+ * how and which part of the model should be saved. Some formats may save
+ * parameters required for training. Some other formats may save model
+ * configurations. The model must be compiled and has allocated memory before
```

Comment:
as there is no interface to allocate memory for the model, please change the requirement of the model state to be compiled/initialized depending on what the API states.

Reply:
Done